src/install.js

```diff
@@ -38,7 +38,8 @@ module.exports = async function install({
     range: defaultTo
   });
 
-  let defaultBlueprintOverride = await module.exports.getBlueprintNameOverride(packageName);
+  let defaultBlueprintOverride = await module.exports.getBlueprintNameOverride(addon, cwd);
```

Comment:
resolvePackage above will turn a url into a local path (by downloading). I think you want `path || packageName` instead of `addon` to avoid double downloading?

Reply:
Switched to `path || packageName`